### DISCUSSION TITLE
【ERROR】2024-07-21 15:45:26,606 - download.py - 未找到下载目录：

### DISCUSSION THREAD

**Discussion by pythonmanGo** [jxxghp/MoviePilot#2595] [问题咨询]
【ERROR】2024-07-21 15:45:26,606 - download.py - 未找到下载目录：

请问用docker下载报错这个是什么原因？已经设置了下载目录并且设置了权限‘’

**Comment by RookiePeter** [comment]
这个有个bug,你下载目录媒体类型仅设置一个全部是不行的
最少要两个,一个电影一个电视剧
